amethyst_config/src/lib.rs

```diff
@@ -107,10 +152,20 @@ where
     }
 
     /// Loads configuration structure from raw bytes.
-    fn load_bytes(bytes: &[u8]) -> Result<Self, ConfigError>;
-
+    fn load_bytes_format(format: ConfigFormat, bytes: &[u8]) -> Result<Self, ConfigError>;
+    
+    /// Writes a configuration structure to a file.
+    fn write_format<P: AsRef<Path>>(
+        &self,
+        format: ConfigFormat,
+        path: P,
+    ) -> Result<(), ConfigError>;
+    
     /// Writes a configuration structure to a file.
-    fn write<P: AsRef<Path>>(&self, path: P) -> Result<(), ConfigError>;
+    #[deprecated(note = "use `write_format` instead")]
+    fn write<P: AsRef<Path>>(&self, format: ConfigFormat, path: P) -> Result<(), ConfigError> {
```

Comment:
API changed, you probably want to drop the `format: ConfigFormat` parameter.

Reply:
That's what I meant to do sorry